drivers/sensor/scd30/Kconfig

```diff
@@ -0,0 +1,30 @@
+
+menuconfig SCD30
+    depends on I2C
+    select SENSIRION_COMMON
+    bool "Sensirion SCD30 CO2 Sensor"
+    default false
+
+if SCD30
+
+comment "SCD30 Options"
+
+config SCD30_MEASUREMENT_INTERVAL
+    int "measurement interval in seconds"
+    default 30
+    range 2 1800
+    help
+      The interval with wich measurement are done.
+      Has to be in the range [2; 1800].
+
+config SCD30_ENABLE_ALTITUDE_COMPENSATION
+    bool "Enable altitude compensation"
```

Comment:
As per the guidelines, please remove `Enable`:
https://docs.zephyrproject.org/latest/guides/build/kconfig/tips.html#prompt-strings

> For a Kconfig symbol that enables a driver/subsystem FOO, consider having just “Foo” as the prompt, instead of “Enable Foo support” or the like.

Also help text is missing.
I can guess what `altitude compensation` means, but what I cannot guess is in which way the measurement is compensated.

I assume it's the CO2 measurement that is compensated, but in which way and what does that mean.